src/TConsole.cpp

```diff
@@ -480,10 +480,9 @@ TConsole::TConsole(Host* pH, ConsoleType type, QWidget* parent)
     splitter->setSizes(sizeList);
 
     mUpperPane->show();
-    mLowerPane->show();
-    mLowerPane->updateScreenView();
-    // timer needed as updateScreenView doesn't seem to finish in time
-    QTimer::singleShot(0, [this]() { mLowerPane->hide(); });
+    // There is never a need to have the lower pane showing from this
+    // constructor as the associated TBuffer must always be empty at this point:
```

Comment:
The comment looks out of place without any context - I'd say remove it. We have Git history to look back upon, right.